app/assets/stylesheets/application/base/_typography.scss

```diff
@@ -14,7 +14,7 @@ h3.history {
 }
 
 .description {
-  font-size: 0.8em;
+  font-size: 1em;
```

Comment:
This probably a good thing, but may be confusing since we use description all over the site and have relied on it being smaller in size... From a usability standpoint, much more readable at 1em though.

Reply:
That's a good point. I'll create a new class for a larger version.